src/components/icon/icons/opaque-circle.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function OpaqueCircle({
+  stroke,
+  fill,
+  strokeWidth,
+  width,
+  height,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+
+  return (
+    <svg width={width} height={height} style={style} {...rest}>
+      <g stroke="none" strokeWidth={strokeWidth} fillRule="evenodd">
```

Comment:
You can probably combine these 2 `<g>` lines, eg. change this line to: `<g stroke="none" strokeWidth={strokeWidth} fillRule="nonzero">` and then remove the line below